lrpar/examples/pretty_errors/src/main.rs

```diff
@@ -239,6 +239,9 @@ fn main() -> ExitCode {
                     .write_parser()
                     .unwrap();
             } else {
+                // Here if running under build.rs/cargo you should use `write` instead of `eprint`
```

Comment:
So one problem with this example has a catch-22 style issue, in build.rs it is giving an example where build.rs fails.
If we include that in the workspace, there is no way to signal to cargo that this is the expected result.
If we do not include it in the workspace, it won't even be compiled though, and could easily bit-rot.

One thing we _could_ do is just include warnings but not errors and include this in build.rs,
but that means that a top-level compile will actually emit warnings I think, but it would resolve some of the issues
with this code trying to act like a build.rs but inside a main.rs, lacking the cargo environment.

Maybe it is best to link from this example a separate repository or a chapter in the book with an _actual_ example as it is intended to be used under cargo?